mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/api/akka/route/v1/JobArtifactsRoute.java

```diff
@@ -85,6 +86,14 @@ public Route constructRoutes() {
                                 // GET - search job artifacts names by prefix
                                 () -> get(this::listJobArtifactsByNameRoute)
                             )
+                        ),
+                        // /api/v1/jobArtifacts/cache
+                        path(
+                            "cache",
+                            () -> pathEndOrSingleSlash(
+                                // POST - cache job artifact on resource cluster
+                                () -> post(this::cacheJobArtifactOnResourceCluster)
+                            )
```

Comment:
I think the ResourceClusterRoute is a more appropriate location for this endpoint.